src/sirius/types.h

```diff
@@ -80,6 +80,21 @@ struct Size {
     int col{0};
 };
 
+struct Point {
+    Point() = default;
```

Comment:
On another point.
**Nowadays**, the more I think about it, the less I'm convinced having a default constructor that zero-initializes numeric objects is a good idea. In that case, we cannot know when the code forgets to declare the object to something meaningful.

IMO, something explicit is better

```C++
// Case 1: no need to pay for something we don't use
point p;
if (foobar) {
    p = f();
} else {
    p = point::zero;
}

// Case2: const is better
const auto p = foobar ? f() : point::zero;

// Case3: let's trust "clang++ -Wall"
// here clang should tell us we forget to assign a meaningful value to p
// unless there is a default constructor that zero-initializes which neutralizes
// clang analysis of uninitialized values.
point p;
if (foobar) p = f();

g(p); 
```